vetiver/handlers/pytorch.py

```diff
@@ -18,8 +19,7 @@ class TorchHandler:
         a trained torch model
     """
     def __init__(self, model, ptype_data):
-        self.model = model
-        self.ptype_data = ptype_data
+        super().__init__(model, ptype_data)
```

Comment:
As it is only calling the superclass, the whole method can be omitted. Same for the `ptype` method.